projects/step-frontend/src/lib/modules/report-nodes/components/call-function-report-node/call-function-report-node.component.html

```diff
@@ -32,11 +32,7 @@
       <ng-container matColumnDef="actions">
         <th mat-header-cell *matHeaderCellDef></th>
         <td mat-cell *matCellDef="let measure">
-          <a
-            [href]="measure | callFunctionReportNodeExternalLink: node"
-            matTooltip="Search in interactive session"
-            target="_blank"
-          >
+          <a (click)="navigateToAnalyticsView(measure)" matTooltip="Search in interactive session" target="_blank">
```

Comment:
`target="_blank"` is redundant here.
When anchor is handled by click, usually we do it in the following way:
```
<a href="#" (click)="navigateToAnalyticsView(measure); false" matTooltip="Search in interactive session">
    <step-icon name="external-link"></step-icon>
</a>
```

Reply:
We actually agreed to have the ;false "hack" as standard behaviour in FE